images/build/cross/variants.yaml

```diff
@@ -53,6 +62,15 @@ variants:
     GO_MAJOR_VERSION: '1.22'
     OS_CODENAME: 'bullseye'
     REVISION: '0'
+  v1.29-go1.23-bullseye:
```

Comment:
I don't think we plan on releasing any more 1.29 patches, so we can likely remove these.  I don't think it will hurt anything and potentially might be useful for downstream folks?

@kubernetes/release-managers, we have [marked 1.29 as EOL now right](https://kubernetes.io/releases/patch-releases/#non-active-branch-history)?

Reply:
Yep we can drop that

Reply:
Sorry, it seemed like that already built the 1.23.6 bits for go runner, etc. I thought this was just filling in the last missing bits for 1.29/1.30

happy to close this in favor of a more correct sequence if you want to take over

Reply:
will do, will start today soon